### PR TITLE
[iOS] in app browsing

### PR DESCRIPTION
**Pull request by thangpham26394**

### CODE REVIEW

File: KrystalWallets/Public/Wallets/WalletManager.swift [hunk 1]
```diff
@@ -109,6 +109,10 @@ public extension WalletManager {
     }
   }
   
+  func emptyAddress() -> KAddress {
+    return KAddress(id: "", walletID: "", addressType: .evm, name: "", addressString: "")
+  }
+
```
Comment (tungnguyen20) [inline KrystalWallets/Public/Wallets/WalletManager.swift:115]: Same here

File: KrystalWallets/Public/Models/Address.swift [hunk 2]
```diff
@@ -18,4 +18,7 @@ public struct KAddress {
     return walletID.isEmpty
   }
   
+  public var isBrowsingWallet: Bool {
+    return id.isEmpty
+  }
```
[the remark this answers is not in the document]
Comment (tungnguyen20) [reply KrystalWallets/Public/Models/Address.swift:23]: Btw, isBrowsingWallet seems not a good name

File: KyberNetwork/KyberNetwork/Screens/Wallet/LandingPage/Coordinator/KNLandingPageCoordinator.swift [hunk 3]
```diff
@@ -4,11 +4,14 @@ import UIKit
 import SafariServices
 import MessageUI
 import KrystalWallets
+import WalletCore
+import RealmSwift
```
Comment (tungnguyen20) [inline KyberNetwork/KyberNetwork/Screens/Wallet/LandingPage/Coordinator/KNLandingPageCoordinator.swift:8]: No need to import them

**Review by MrQKyber — APPROVED** [review]
lgtm

**Pull request merged**